lab_3_generate_by_ngrams/main.py

```diff
@@ -208,6 +299,19 @@ def build(self) -> int:
         In case of corrupt input arguments or methods used return None,
         1 is returned
         """
+        if not isinstance(self._encoded_corpus, tuple) or \
+                len(self._encoded_corpus) == 0:
```

Comment:
Please check it a better way

Reply:
Here and everywhere else: `== 0` is a redundancy

Reply:
Fixed